Assets/MixedRealityToolkit/_Core/DataProviders/SpatialObservers/BaseMixedRealitySurfaceObserver.cs

```diff
@@ -37,6 +38,23 @@ protected BaseMixedRealitySurfaceObserver(string name, uint priority, BaseMixedR
             PlatformSurfaceMaterial = profile.PlatformSurfaceMaterial;
         }
 
+        /// <inheritdoc />
+        public override int PhysicsLayer
```

Comment:
It seems that, since we are providing an override property, the override should report the base layer if not configured and should not replace the value of the base property (it should remain easy to access)

The implementation is also different from the mesh layer override

```
        public override int PhysicsLayer => MeshPhysicsLayerOverride >= 0 ? MeshPhysicsLayerOverride : base.PhysicsLayer;
```

Reply:
Suggest something similar to the following (for both current overrides)

```
public int MeshPhysicsLayerOverride => (meshPhysicsLayerOverride > -1) ? meshPhysicsLayerOverride : base.PhysicsLayer;
```

Rather than overriding the base property.

Reply:
I don't see the difference.

Reply:
I think this pattern makes sense to me. This way, everything can keep referring to the generic `PhysicsLayer`, which will return the override in cases where it's set.

The logic does appear different to the mesh implementation though.

Reply:
I'll update the surfaces observer to match the mesh observer behavor when i open my PR to restore the spare object logic. Hopefully that will help clarify